src/WebClient.ts

```diff
@@ -121,7 +121,11 @@ export class WebClient extends EventEmitter {
 
     // The following thunk is the actual implementation for this method. It is wrapped so that it can be adapted for
     // different executions below.
-    const implementation = () => {
+    const implementation = async () => {
+
+      if (typeof options === 'string' || typeof options === 'number' || typeof options === 'boolean') {
```

Comment:
Could you check for `is an object` rather than `is not x, or x, or x`?

Reply:
`typeof null === 'object'` so that would pass this check. that being said, it passes this check right now too. 🤔

Reply:
im afraid of weird javascript quirks, and this is more explicit regardless. `null`s would still pass this check, but `null` never has been a valid argument value, so i'm not too afraid of that case. i stand by this condition.